src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/hosts/hosts.component.ts

```diff
@@ -125,6 +129,16 @@ export class HostsComponent extends ListWithDetails implements OnDestroy, OnInit
   ) {
     super();
     this.permissions = this.authStorageService.getPermissions();
+    this.expandClusterActions = [
+      {
+        name: this.actionLabels.EXPAND_CLUSTER,
+        permission: 'create',
+        icon: Icons.exit,
```

Comment:
just saw this icon, probably this should be replaced with something more meaningful for an expansion?

Reply:
I put that bcs it will redirect to a different screen, but yes we can change it, do you have any suggesstion?

Reply:
https://forkaweso.me/Fork-Awesome/icon/expand/?